src/app/_components/Header/Previews/ChatPreviews.tsx

```diff
@@ -0,0 +1,61 @@
+import { useQuery } from '@tanstack/react-query';
+import { CloseSVG } from '@/icons/svg';
+import { getOpponentInfo } from '@/lib/apis/chatApi';
+import ProfileImg from '@/components/Profile/ProfileImage';
+import { userType } from '@/types/auth';
+import { Chat } from '@/types/chat';
+
+interface ChatPreviewProps {
+  chat: Chat;
+  opponentType: userType;
+  closeChatPreview: () => void;
+}
+
+const ChatPreviews = ({
+  chat,
+  opponentType,
+  closeChatPreview,
+}: ChatPreviewProps) => {
+  const opponentId = chat.sender[
+    opponentType === 'lecturer' ? 'lecturerId' : 'userId'
+  ] as number;
+
+  const { data, isLoading } = useQuery({
+    queryKey: ['opponentProfile', opponentType, opponentId],
+    queryFn: () => getOpponentInfo(opponentType, opponentId),
+    staleTime: Infinity,
+  });
+
+  const closePreviewHandler = (event: React.MouseEvent<HTMLDivElement>) => {
+    event.stopPropagation();
+    closeChatPreview();
+  };
+
+  return (
+    <>
+      <div className="relative grid grid-cols-[auto_1fr] items-center">
+        {isLoading ? (
+          <>
+            <div className="mr-3 size-[34px] flex-shrink-0 animate-pulse rounded-full bg-gray-500" />
+            <div className="h-4 w-14 animate-pulse bg-gray-500" />
+          </>
+        ) : (
+          <>
+            <ProfileImg src={data?.profileImg} size="small" />
+            <div className="flex-grow truncate text-left font-bold">
```

Comment:
```suggestion
            <p className="flex-grow truncate text-left font-bold">
```
text라서 p가 좀 더 적절할 거 같은....의견,,,

Reply:
맞아맞아 고마워 👍